### PR TITLE
Add Server settings

### PR DESCRIPTION
**Pull request by jeffwang0516**
1. Using nginx & uwsgi
2. https server
3. Settings files in new folder /server-settings/
4. Readme updated

### CODE REVIEW

File: server-settings/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    # the domain name it will serve for
+    server_name smartcampus.csie.ncku.edu.tw    # substitute your machine's IP address or FQDN
+    charset     utf-8;
+
+    #Max upload size
+    client_max_body_size 75M;   # adjust to taste
+
+    # Django media
+    location /media  {
+                alias /home/rapirent/smart_campus/smart_campus/smart_campus/media;      # your Django project's media files
```
Comment (Lee-W) [inline server-settings/nginx.conf]: Is it possible to use `$HOME` instead of `/home/rapirent`?

Comment (jeffwang0516) [reply server-settings/nginx.conf]: Not working

Comment (Lee-W) [reply server-settings/nginx.conf]: I knew.
Hmmm...
I'll search whether there exists a better way.

Comment (rapirent) [reply server-settings/nginx.conf]: I think "in the production" we should change the MEDIA_URL from BASE_DIR to `/var/` like `/var/www/example.com/media/`, and add an line ` MEDIA_URL = "/var/www/example.com/media/"` to the setting/production.py

ref: https://docs.djangoproject.com/en/1.11/ref/settings/

Comment (Lee-W) [reply server-settings/nginx.conf]: Great Job

**Pull request merged**